README.md

```diff
@@ -93,15 +93,35 @@ To enable Metadata Reflection:
   ```html
   <script src="/bower_components/reflect-metadata/Reflect.js"></script>
   ```
+  
+To enable the `@listen` decorator:
```

Comment:
Maybe add an Event Listeners heading, so it's not under the Metadata Reflection heading.